src/slider/slider.scss

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-baseline; // 11px
+
+@mixin iui-slider-component-container {
+  display: flex;
+
+  .iui-slider-min,
+  .iui-slider-max {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
```

Comment:
Here we could use a longer class name to reduce nesting (and specificity).

Reply:
Seems your react implementation requires the user to apply this class. https://github.com/iTwin/iTwinUI-react/pull/197/files#diff-73f4c3c7fc71dbb9d680fddeb2ede7bbe574448f82da3ba5e3e4f6cc197c91deR27

I would have expected that you clone it and apply it yourself, which is why I suggested the longer name.

An alternative is to target `svg` instead.

Reply:
@mayank99 - Jon added the iui-icon rule and I added the class in the storybook example so icon was properly styled. I am unsure what you mean by cloning or targeting svg.

Reply:
cloning:
```tsx
React.cloneElement(startIcon, { className: cx('iui-icon', startIcon.props.className) }
```

targeting svg:
```scss
svg {
  @include iui-icons-default();
  ...
}
```